<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>441</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>441</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QHeaderView::section 
{
 	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #fff, stop:1 #333);
	color: #333;
	height: 1.54em;
}

QMainWindow#MainWindow * 
{
	color: rgb(187,187,187);
    background: rgb(68,68,68);
}

QMainWindow#MainWindow
{
    color: rgb(187,187,187);
	background: rgb(68,68,68);
	/*border: 0;*/
}

QMenuBar 
{
	color: rgb(187,187,187);
}
QMenuBar::item 
{
 	background: transparent;
 	color: rgb(187,187,187);
 	/*border: 0;*/
}

QMenuBar::item:selected 
{
	background: rgb(82,133,166);

}
QMenu::item:selected 
{
	background: rgb(82,133,166);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Microsoft Sans Serif</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	color:  rgb(187, 187, 187);
	background-color: rgb(68, 68, 68);
}

QLineEdit
{
	border: none;
	border-width: 2px;
	color:  rgb(187, 187, 187);
	background-color: rgb(43, 43, 43);
	padding: 2px;	
}

QPlainTextEdit
{
	border: none;
	color:  rgb(187, 187, 187);
	background-color: rgb(43, 43, 43);	
}

QPushButton
{
	border-style: outset;
	color:  rgb(187, 187, 187);
	background-color: rgb(93, 93, 93);
	padding: 5px;
}

QPushButton:hover
{
   background-color: rgb(112, 112, 112);
}

QSpinBox
{
    border: none;
	color:  rgb(187, 187, 187);
	background-color: rgb(43, 43, 43);
}

QSpinBox:disabled
{
	color:  rgb(107, 107, 107);
	background-color: rgb(63, 63, 63);
}

QLabel:disabled
{
	color:  rgb(107, 107, 107);
}

QLineEdit:disabled
{
	color:  rgb(107, 107, 107);
	background-color: rgb(63, 63, 63);
}	</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="l_exportLoc">
          <property name="font">
           <font>
            <family>Microsoft Sans Serif</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Export Location</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="btn_export">
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="text">
             <string> Directory </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="s_exportLoc">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="l_refRig">
          <property name="font">
           <font>
            <family>Microsoft Sans Serif</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Reference Rig</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QPushButton" name="btn_refRig">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="text">
             <string>      File      </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="s_refRig">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="l_shftKeyData">
          <property name="font">
           <font>
            <family>Microsoft Sans Serif</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Shift Key Data</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="i_startFrame">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <number>-999999</number>
            </property>
            <property name="maximum">
             <number>999999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="l_newStartFrame">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="text">
             <string>New Starting Frame</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="b_shiftKeys">
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="text">
             <string>Shift Keys</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="l_topContNode">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="text">
             <string>Top Control Node    </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="s_contNode">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="l_singleFile">
          <property name="font">
           <font>
            <family>Microsoft Sans Serif</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Single File</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="btn_singleFile">
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="text">
             <string>      File      </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="s_singleFileLoc">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="b_singleExportFBX">
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="text">
             <string>Export FBX</string>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_singleConvert">
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
             </font>
            </property>
            <property name="text">
             <string>     Convert     </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="l_batch">
            <property name="font">
             <font>
              <family>Microsoft Sans Serif</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Batch</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QPushButton" name="btn_batchFile">
              <property name="font">
               <font>
                <family>Microsoft Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string> Directory </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="s_batchDirLoc">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Microsoft Sans Serif</family>
               </font>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="1">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="b_batchExportFBX">
              <property name="font">
               <font>
                <family>Microsoft Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>Export FBX</string>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="btn_batchConvert">
              <property name="font">
               <font>
                <family>Microsoft Sans Serif</family>
               </font>
              </property>
              <property name="text">
               <string>     Convert     </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="actionEnglish"/>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChinese">
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>b_shiftKeys</sender>
   <signal>toggled(bool)</signal>
   <receiver>i_startFrame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_shiftKeys</sender>
   <signal>toggled(bool)</signal>
   <receiver>l_newStartFrame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_shiftKeys</sender>
   <signal>toggled(bool)</signal>
   <receiver>s_contNode</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_shiftKeys</sender>
   <signal>toggled(bool)</signal>
   <receiver>l_topContNode</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
